db/schema.rb

```diff
@@ -142,4 +142,7 @@
 
   add_index "versions", ["item_type", "item_id"], name: "index_versions_on_item_type_and_item_id", using: :btree
 
+  add_index "views", ["email"], name: "index_views_on_email", unique: true, using: :btree
```

Comment:
Travis build won't pass until this indexing code is removed, since the schema doesn't define the `views` table